nautobot_golden_config/forms.py

```diff
@@ -480,27 +480,26 @@ class ConfigPlanForm(NautobotModelForm):
             "You can also reference the device object with <code>obj</code>.<br>"
             "For example: <code>hostname {{ obj.name }}</code> or <code>ip address {{ obj.primary_ip4.host }}</code>"
         ),
-        required=True,
     )
 
-    tenant_group = utilities_forms.DynamicModelMultipleChoiceField(queryset=TenantGroup.objects.all(), required=False)
-    tenant = utilities_forms.DynamicModelMultipleChoiceField(queryset=Tenant.objects.all(), required=False)
+    tenant_group = utilities_forms.DynamicModelMultipleChoiceField(queryset=TenantGroup.objects.all())
```

Comment:
more just curious why required=False is being removed? Is that a default? If so; does it really hurt to be pointed?

Reply:
It would be part of best practices (I believe). It is coming from the model otherwise, and doing this leaves the chance for the model and the form to disagree. So my logic would be, it is not set or there is a comment saying why it is set. 

I am learning all this as well, just something I realized with nautobot-pylint recommendations on CharField and figure let's just get this right from the start.

Reply:
thanks, makes sense.

Reply:
I was partially (or maybe mostly) incorrect, documented here: https://github.com/nautobot/nautobot-plugin-golden-config/pull/577#discussion_r1322939551